packages/serverless/src/integrations/aws_lambda.ts

```diff
@@ -0,0 +1,235 @@
+     * @hidden
+     */
+    function timeOutError(configuredTime: number): void {
+      setTimeout(() => {
+        /**
+         * setting parameters in scope which will be displayed as additional data in Sentry dashboard
+         */
+        setParameters();
+
+        const convertTimeToSeconds = Math.floor(configuredTime / millisToSeconds);
+
+        const error = new Error(
+          `WARNING : Function is expected to get timed out. Configured timeout duration = ${
+            convertTimeToSeconds + 1
+          } seconds.`,
+        );
+
+        /** capturing the exception and re-directing it to the Sentry Dashboard */
+        hub.captureException(error);
+        Sentry.flush(2000);
+      }, configuredTime);
+    }
+
+    /**
+     * This function sets Additional Runtime Data which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    function setAdditionalRuntimeData(scope: Scope): void {
+      scope.setContext('Runtime', {
+        Name: nodeJs,
+        // global.process.version return the version of node
+        Version: global.process.version,
+      });
+    }
+
+    /**
+     * This function sets Additional Lambda Parameters which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    function setAdditionalLambdaParameters(scope: Scope): void {
+      const remainingTimeInMillisecond: number = setupOnceContext.awsContext.getRemainingTimeInMillis() as number;
+      const executionTime: number = setupOnceContext.configuredTimeInMilliseconds - remainingTimeInMillisecond;
+
+      scope.setExtra('lambda', {
+        aws_request_id: setupOnceContext.awsContext.awsRequestId,
+        function_name: setupOnceContext.awsContext.functionName,
+        function_version: setupOnceContext.awsContext.functionVersion,
+        invoked_function_arn: setupOnceContext.awsContext.invokedFunctionArn,
+        execution_duration_in_millis: executionTime,
+        remaining_time_in_millis: setupOnceContext.awsContext.getRemainingTimeInMillis(),
+      });
+    }
+
+    /**
+     * This function sets Cloud Watch Logs data which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    function setCloudwatchLogsData(scope: Scope): void {
+      scope.setExtra('cloudwatch logs', {
+        log_group: setupOnceContext.awsContext.logGroupName,
+        log_stream: setupOnceContext.awsContext.logStreamName,
+        /**
+         * process.env.AWS_REGION -  this parameters given the AWS region
+         * process.env.AWS_LAMBDA_FUNCTION_NAME - this parameter provides the AWS Lambda Function name
+         */
+        url: `https://${process.env.AWS_REGION}.console.aws.amazon.com/cloudwatch/home?region=${process.env.AWS_REGION}#logsV2:log-groups/log-group/$252Faws$252Flambda$252F${process.env.AWS_LAMBDA_FUNCTION_NAME}`,
```

Comment:
I'd extract this somewhere at the top of the integration as a const function. It shouldn't be "this" hardcoded here as it might change.

Reply:
made this change in the new pull request.